### PR TITLE
Add README.md, fix minor corrections.

### PR DESCRIPTION
**Pull request by terabytesoftw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<p align="center">
+    <a href="https://github.com/yiisoft" target="_blank">
+        <img src="https://avatars0.githubusercontent.com/u/993323" height="100px">
+    </a>
+    <h1 align="center">Yii Actions</h1>
```
Comment (vjik) [inline README.md:5]:
```suggestion
    <h1 align="center">Yii GitHub Actions</h1>
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    <br>
+</p>
+
+
+## General usage
+
+This package allows us to reuse github actions, which simplifies version management.
```
Comment (vjik) [inline README.md:12]:
```suggestion
This package allows us to reuse GitHub actions, which simplifies version management.
```

File: README.md (hunk 1 above)
Comment (terabytesoftw) [reply README.md:5]: done.

File: README.md (hunk 2 above)
Comment (terabytesoftw) [reply README.md:12]: done.

**Review by vjik — APPROVED** [review]

**Pull request merged**